Detections/ProofpointPOD/ProofpointPODMultipleProtectedEmailsToUnknownRecipient.yaml

```diff
@@ -42,5 +42,5 @@ entityMappings:
     fieldMappings:
       - identifier: FullName
         columnName: AccountCustomEntity
-version: 1.0.0
-kind: Scheduled
\ No newline at end of file
+version: 1.0.1

```

Comment:
version should be updated to 1.1.0. Same in other places.
On the logic of the change, please ask for @shainw  to review.
Thanks!

Reply:
Updated @Amitbergman .
Request @shainw to review for logic change.